docs/3x/PT_StaticQuant.md

```diff
@@ -0,0 +1,64 @@
+PyTorch Static Quantization
+========================================
+
+1. [Introduction](#introduction)
+2. [Get Started](#get-started) \
```

Comment:
are we missed on section for TorchDynamc path here?  @yiliu30 @violetch24

Reply:
Yes，pt2e for static quant will be delay to tmr.